inclusive-design/wecount.inclusivedesign.ca
# feat: use Avenir for logotype (fixes #145)

## Description

Update brand logotype to use an SVG of the correct typeface.

## Steps to test

1. Visit deploy preview.
2. Compare logo type to Figma.

**Expected behavior:** Typeface matches.

## Additional information

I implemented this using an SVG instead of a web font because we only need seven characters and because loading a webfont for this would be inefficient.

## Related issues

Resolves #145.
